### DISCUSSION TITLE
DAI 2023-2024 - Practical work 4 - Thomas Vuilleumier, Sebastian Diaz, Arthur Menétrey and Lionel Pollien

### DISCUSSION THREAD

**Discussion by Cobora2001** [heig-vd-dai-course/heig-vd-dai-course#185] [Show and tell]
Here is our group for the Practical work 4
- Thomas Vuilleumier (@Cobora2001)
- Sebastian Diaz (@MolinaElGringo)
- Arthur Menétrey (@Xenogix)
- Lionel Pollien (@poLionel)

Here is the repository we will use: [Practical Work 4](https://github.com/Cobora2001/DAI_Practical_Work_4)

We don't yet know what we will do.

We have sent the email regarding the acquisition of the virtual ressources (sent by Sebastian).

**Comment by Xenogix** [comment]
You can find our last commit [here (hash 2d836aa)](https://github.com/Cobora2001/DAI_Practical_Work_4/commit/2d836aa4616f9b185f3fea210670f54c865ad31a)

Our docker image can be downloaded [here](https://github.com/Cobora2001/DAI_Practical_Work_4/pkgs/container/dai_practical_work)
